webrender/src/device/gfx.rs

```diff
@@ -715,15 +715,12 @@ impl<B: hal::Backend> Buffer<B> {
         };
         let requirements = unsafe { device.get_buffer_requirements(&buffer) };
 
-        let alignment = std::cmp::max(requirements.alignment, (alignment_mask + 1) as u64);
-        let mask = alignment - 1;
-        let size = (requirements.size + mask) & !mask;
         let memory_block = heaps.allocate(
             device,
             requirements.type_mask as u32,
             memory_usage,
-            size,
-            alignment,
+            requirements.size,
+            std::cmp::max(requirements.alignment, (alignment_mask + 1) as u64),
```

Comment:
Is mask supposed to be used like that?
And not like `(requirements.alignment - 1) | (alignment_mask as u64) + 1`?

Reply:
Yes, that would make more sense.